### DISCUSSION TITLE
text conditioning

### DISCUSSION THREAD

**Discussion by nikuson** [archinetai/audio-diffusion-pytorch#10] [Ideas]
I would like to understand how to do something like Glide in audio diffusion.

**Comment by flavioschneider** [comment]
It's not yet possible to do without changing the model. Text conditioning is currently a work in progress.

**Comment by jbmaxwell** [comment]
I'm training on the text conditioning model, using a tweaked `diff_textcond_test.yaml` file, and I'm just wondering how I might make it stereo, rather than mono? I've been looking at other stereo (channels = 2) examples in `exp`, but I'm pretty certain I'll smash things to a buh-zillion pieces trying to just hack it... :)

Any tips appreciated.
